lib/oca-epak/oep/client.rb

```diff
@@ -17,10 +17,12 @@ def initialize(username, password)
       # @return [String] HTML
       def get_html_de_etiquetas_por_orden_or_numero_envio(opts = {})
         method = :get_html_de_etiquetas_por_orden_or_numero_envio
-        opts = { "idOrdenRetiro" => opts[:id_orden_retiro],
-                 "nroEnvio" => opts[:nro_envio] }
+        opts = {
+          "idOrdenRetiro" => opts[:id_orden_retiro],
```

Comment:
do you mean with an "I" instead of an "i"? or a missed something, based on the XML for the request it's ok with the small caps "i"
```xml
<x:Envelope xmlns:x="http://schemas.xmlsoap.org/soap/envelope/" xmlns:oca="#Oca_Express_Pak">
    <x:Header/>
    <x:Body>
        <oca:GetHtmlDeEtiquetasPorOrdenOrNumeroEnvio>
            <oca:idOrdenRetiro>0</oca:idOrdenRetiro>
            <oca:nroEnvio>?</oca:nroEnvio>
        </oca:GetHtmlDeEtiquetasPorOrdenOrNumeroEnvio>
    </x:Body>
</x:Envelope>
```